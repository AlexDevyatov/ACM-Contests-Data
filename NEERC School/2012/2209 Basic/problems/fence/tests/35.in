9
-90 -90
52 -90
87 -86
87 -36
82 68
-9 86
-60 77
-83 46
-90 -35
-58 -53
