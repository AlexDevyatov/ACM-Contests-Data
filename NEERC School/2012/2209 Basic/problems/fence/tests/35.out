10
-90 -90
52 -90
-58 -53
87 -86
87 -36
82 68
-9 86
-60 77
-83 46
-90 -35
